code/modules/reagents/chemistry/reagents/food.dm

```diff
@@ -747,16 +795,13 @@
 			M.Stun(rand(2,5))
 	..()
 
-/datum/reagent/msg/reaction_mob(mob/living/M, method=TOUCH, volume)
-	if(method == INGEST)
-		to_chat(M, "<span class='notice'>That tasted amazing!</span>")
-
 /datum/reagent/cholesterol
 	name = "cholesterol"
 	id = "cholesterol"
 	description = "Pure cholesterol. Probably not very good for you."
 	reagent_state = LIQUID
 	color = "#FFFAC8"
+	taste_message = "heart disease"
```

Comment:
If too much cholesterol in SS13 can actually give people a heart attack I'd say to change this to "heart-attack"